site/data/docs/main-toc.yml

```diff
@@ -39,6 +39,8 @@ toc:
         url: /config/annotations
       - page: API Reference
         url: /config/api
+      - page: Access logging
+        url: /config/access-logging
```

Comment:
I'd move this up to just above "Annotations Reference" to leave the two reference pages at the bottom of this TOC section. Separately (not needed for this PR) we might want to move them into their own "Reference" section.

Reply:
Fixed!